docs/source/models/intervention_models/maternal_supplementation/index.rst

```diff
@@ -484,6 +484,155 @@ In the baseline scenario, the exposure distribution of birthweight (mean birthwe
 
 If birthweight exposures are stratified by supplementation regimen and maternal nourishment strata, then birthweight differences between regimens should match the effect sizes within a given maternal nourishment exposure strata.
 
+Gestational age
++++++++++++++++++++
+
+.. note::
+
+  This outcome has been added in June of 2023, and will be first incorporated into the :ref:`nutrition optimization <2021_concept_model_vivarium_nutrition_optimization>` model and has not been included in the :ref:`IV iron <2019_concept_model_vivarium_iv_iron>` or :ref:`BEP <2017_concept_model_vivarium_gates_bep>` simulations. 
+
+Research background
+~~~~~~~~~~~~~~~~~~~
+
+The antenatal supplementation products affect child gestational age at birth exposures, :ref:`which are documented here <2019_risk_exposure_lbwsg>`. While we measure LBWSG exposures at the continuous level (including a joint birth weight and gestational age at birth value), the literature tends to report the effect of antenatal supplementation products on gestational age at birth in terms of a relative risk of preterm birth (less than 37 weeks gestational age at birth) or very preterm birth (less than 32 weeks gestational age at birth), which are summarized in the table below.
+
+.. list-table:: Dichotomous effect of antenatal supplementation on preterm birth
+  :header-rows: 1
+
+  * - Product
+    - Relative to 
+    - Outcome
+    - Value
+    - Source
+  * - IFA
+    - No IFA
+    - Preterm birth, <37 weeks
+    - OR = 0.9 (95% CI: 0.86, 0.95)
+    - [Li-et-al-2019-antenatal-supplementation]_
+  * - MMS
+    - IFA
+    - Preterm birth, <37 weeks
+    - RR = 0.95 (95% CI: 0.90, 1.01)
+    - [Keats-et-al-2019-maternal-supplementation]_
+  * - MMS
+    - IFA
+    - Very preterm birth, <32 weeks
+    - RR = 0.81 (95% CI: 0.71, 0.93)
+    - [Keats-et-al-2019-maternal-supplementation]_
+  * - BEP
+    - MMS
+    - Preterm birth, <37 weeks
+    - 0.96 (0.80, 1.16); assume no effect
+    - [Ota-et-al-2015]_
+
+In order to make these effects compatible with our continuous exposure modeling strategy for LBWSG, we have converted these relative risks of dichotomous outcomes to continuous gestational age "shifts" that result in preterm (and very preterm, if applicable) birth prevalence that replicates the appropriate dichotomous measure of effect. 
+
+The methodology for this conversion was inspired in part by the methodology of the air pollution GBD team in calculating the impact of the risk on LWBSG exposures. As with GBD, we assume that these shifts are independent of any shifts in birth weight. However, rather than implement the conversion using the LBWSG exposure distribution specific to the individual study included in a meta-analysis of the overall effect to find a global shift as GBD did, we used the meta-analyzed global relative risks and applied the conversion for each of our modeled locations, resulting in location-specific continuous shifts that replicate the global dichotomous effect.
+
```

Comment:
```suggestion
In the case of birth weight shifts, it may be more accurate to assume that the relative risks are generalizable across locations.

```